backend/dtos/dtos.go

```diff
@@ -47,12 +47,31 @@ type Finding struct {
 	OccurredTo    *time.Time `json:"occurredTo"`
 }
 
+type TopContrib struct {
+	Slug        string `db:"slug" json:"slug"`
+	OperationID int64  `db:"operation_id" json:"operationId"`
```

Comment:
If possible, I think you should change this to `OperationSlug string`. I think we're intentionally hiding IDs

Reply:
yeah I'm just using this to group these with the operations, so I can group them by slug instead

Reply:
oh oops I may have to do that by ID, and just drop the ID before I add to the response body or something

Reply:
took me forever to get this working correctly, but removed the IDs